code/game/objects/items/plantable_flags.dm

```diff
@@ -1,60 +1,225 @@
-/obj/item/flag_base
-	name = "basic flag"
-	desc = "It's a one time use flag built into a telescoping pole ripe for planting."
-	icon = 'icons/obj/items/plantable_flag.dmi'
-	icon_state = "flag_collapsed"
-	force = 3
-	throwforce = 2
+///Range of the aura
+#define FLAG_AURA_RANGE 10
+///Range of the aura when deployed
+#define FLAG_AURA_DEPLOYED_RANGE 14
+///The range in tiles which the flag makes people warcry
+#define FLAG_WARCRY_RANGE 5
+///Strength of the aura
+#define FLAG_AURA_STRENGTH 3
+///Penalty aura for losing or otherwise disgracing the flag
+#define LOST_FLAG_AURA_STRENGTH -2
+
+/obj/item/plantable_flag
+	name = "\improper TerraGov flag"
+	desc = "A flag bearing the symbol of TerraGov. It flutters in the breeze heroically. This one looks ready to be planted into the ground."
+	icon = 'icons/obj/items/flags/plantable_flag_large.dmi'
+	icon_state = "flag_tgmc"
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	worn_icon_list = list(
+		slot_l_hand_str = 'icons/mob/inhands/items/large_flag_left.dmi',
+		slot_r_hand_str = 'icons/mob/inhands/items/large_flag_right.dmi',
+	)
+	w_class = WEIGHT_CLASS_HUGE
+	force = 55
+	attack_speed = 15
+	attack_verb = list("stabbed", "thrust", "smashed", "thumped", "bashed", "attacked", "clubbed", "speared", "jabbed", "torn", "gored")
+	sharp = IS_SHARP_ITEM_BIG
 	throw_speed = 1
-	throw_range = 4
-	w_class = WEIGHT_CLASS_SMALL
-	var/is_collapsed = TRUE
-	var/country_name = "TGMC" //presume it is prefaced with 'the'
+	throw_range = 2
+	soft_armor = list(MELEE = 50, BULLET = 50, LASER = 50, ENERGY = 50, BOMB = 50, BIO = 100, FIRE = 50, ACID = 50)
+	///The faction this belongs to
+	var/faction = FACTION_TERRAGOV
+	///Aura emitter
+	var/datum/aura_bearer/current_aura
+
+	var/turf/origin_point
+
+/obj/item/plantable_flag/Initialize(mapload)
+	. = ..()
+	RegisterSignal(SSdcs, COMSIG_GLOB_CAMPAIGN_MISSION_ENDED, PROC_REF(mission_end))
+	origin_point = get_turf(src)
+	if(isturf(loc))
+		item_flags |= DEPLOY_ON_INITIALIZE
+	AddComponent(/datum/component/deployable_item, /obj/structure/plantable_flag, 1 SECONDS, 3 SECONDS)
+	AddComponent(/datum/component/shield, SHIELD_PURE_BLOCKING, list(MELEE = 35, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 0, FIRE = 0, ACID = 0))
```

Comment:
Shield?

Reply:
ye? Its taking up a hand, and will debuff your whole team the instant you drop it, so it has some melee block chance to help them not be fodder

Reply:
ehh ok @Kuro020 fyi